docs/user-guide/src/developer_environment/tryit.md

```diff
@@ -50,6 +50,30 @@ For execution with VMs
 - Save and exit
 - Manually **enable nested virtualization** if you don't have it enabled in your VM
 
+```console
+  # To enable nested virtualization 
+  # On Centos 9 streams (other distros may vary)
+  # check the current setting
+  cat /sys/module/kvm_intel/parameters/nested 
```

Comment:
```suggestion
  $ sudo cat /sys/module/kvm_intel/parameters/nested 
```
Since we're showing command and output, we should display prompt and add sudo where necessary.

Reply:
Yes absolutely, thanks for the review - done :)